core/domain/state_domain.py

```diff
@@ -1063,6 +1063,21 @@ def from_dict(cls, recorded_voiceovers_dict):
 
         return cls(voiceovers_mapping)
 
+    def has_complete_voiceover(self, language_code):
+        """Whether there exist complete voiceover in the given language.
+
+        Args:
+            language_code: str. The code of the language.
+
+        Returns:
+            bool. Returns true if the object has complete voiceover in the give
+            language.
+        """
+        for content_id in self.voiceovers_mapping:
```

Comment:
Is every content id present in voiceover_mapping? Or is it only added there when a voiceover in some language is added for that content_id?

Reply:
All the content_ids are present in the voiceover_mapping dict.